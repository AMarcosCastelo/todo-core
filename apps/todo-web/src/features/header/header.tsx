import { Navbar, Text } from '@nextui-org/react';

export default function Header() {
  return (
    <Navbar isBordered>
      <Navbar.Brand>
        <Text>My Todo-s</Text>
      </Navbar.Brand>
    </Navbar>
  );
}
