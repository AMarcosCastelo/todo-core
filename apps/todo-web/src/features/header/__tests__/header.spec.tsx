import { baseRender } from '../../../utils/testing';
import { Header } from '../header';

describe('Todo form', () => {
  it('should render correctly', () => {
    const screen = baseRender(<Header />);

    expect(screen.container).toMatchSnapshot();
  });
});
