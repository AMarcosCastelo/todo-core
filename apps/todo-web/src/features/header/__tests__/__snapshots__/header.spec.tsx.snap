// Vitest Snapshot v1

exports[`Todo form > should render correctly 1`] = `
<div>
  <div
    data-overlay-container="true"
  >
    <nav
      class="nextui-c-hhqfap nextui-c-hhqfap-gaIaSO-variant-static nextui-c-hhqfap-beFEYN-maxWidth-lg nextui-c-hhqfap-hemDFs-isBordered-true nextui-c-hhqfap-kAwfsS-disableShadow-false nextui-c-hhqfap-gqZKcP-disableBlur-false nextui-c-hhqfap-iPJLV-css nextui-navbar"
    >
      <div
        class="nextui-c-egxOLa nextui-navbar-container"
      >
        <span
          class="nextui-c-fyNvBR nextui-c-PJLV nextui-navbar-brand"
        >
          <svg
            class="lucide lucide-list-checks"
            fill="none"
            height="40"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="40"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="10"
              x2="21"
              y1="6"
              y2="6"
            />
            <line
              x1="10"
              x2="21"
              y1="12"
              y2="12"
            />
            <line
              x1="10"
              x2="21"
              y1="18"
              y2="18"
            />
            <polyline
              points="3 6 4 7 6 5"
            />
            <polyline
              points="3 12 4 13 6 11"
            />
            <polyline
              points="3 18 4 19 6 17"
            />
          </svg>
          <span
            aria-hidden="true"
            class="nextui-c-gNVTSf nextui-c-gNVTSf-hakyQ-inline-false nextui-c-gNVTSf-igvLppd-css"
          />
          <p
            class="nextui-c-PJLV nextui-c-PJLV-ifpQFrU-css"
          >
            My Todo-s
          </p>
        </span>
      </div>
    </nav>
  </div>
</div>
`;
